trackr-app/app/src/main/java/com/trackr/trackr_app/database/EventDao.kt

```diff
@@ -25,6 +25,9 @@ interface EventDao {
     @Query("SELECT * FROM `event-table` WHERE id = :id")
     suspend fun getById(id: String): TrackrEvent
 
+    @Query("SELECT * FROM `event-table` WHERE person_id = :personId")
+    fun getByPersonId(personId: String): Flow<List<TrackrEvent>>
```

Comment:
you should retrieve the person rather than the event.

SELECT * FROM `person-table` WHERE person_id = :personId 

in the person dao